src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/aggregators/PersistableSlidingWindow.java

```diff
@@ -92,6 +92,7 @@ protected synchronized void write() throws IOException {
     }
     // write to temporary file
     writer.flush();
+    writer.close();
```

Comment:
Undo this change and rebase, you'll get the try with resource version

Reply:
yup, done now.